prob2.java

```diff
@@ -0,0 +1,93 @@
+// Time Complexity : O(n)
```

Comment:
Kindly explain your complexity. Also include the length of sentence as well in complexity